docs/updating/nim-migration/online-builder.md

```diff
@@ -0,0 +1,164 @@
+---
+category: nim-migration
+order: 30
+menu-title: Migrating from legacy Online Builder
+meta-title: Migrating from the legacy Online Builder to new installation methods | CKEditor 5 documentation
+meta-description: Learn how to upgrade build from the legacy Online Builder to new installation methods.
+modified_at: 2024-06-25
+---
+
+# Migrating from legacy Online Builder
+
+There are three installation methods you can migrate to from the legacy Online Builder. The best option for you depends on whether you just want an out-of-the-box browser build, or if you want a customized and optimized build.
+
+The npm package is the most flexible and powerful way to install CKEditor&nbsp;5. It allows you to create a custom build of the editor with only the features you need, thus significantly reducing the final size of the build. However, you will need a JavaScript bundler or meta-framework to create such a build.
+
+If you do not want to have a build process, you can either use our CDN build or download the ZIP archive. Both of these include the editor and all plugins, so you can use all the features of CKEditor&nbsp;5 without setting up a build process.
+
+## CDN build
+
+The CDN build is a good option if you want to quickly add CKEditor&nbsp;5 to your website without having to install any dependencies or set up a build process. We recommend using our new interactive [Builder](https://ckeditor.com/ckeditor-5/builder?redirect=docs) to customize the build to your needs. Then, in the `Installation` section of the Builder you can select the `Cloud (CDN)` option to learn how to add the editor to your website.
+
+## ZIP archive
+
+If you do not want to have a build process or use our CDN build, you can download the ZIP archive from the [CKEditor&nbsp;5 download page](https://ckeditor.com/ckeditor-5/download/#zip). This archive contains the editor build with all of its plugins, which you can extract and include on your website.
+
+## npm package
+
+If you decide to use the npm package, you can either use our new interactive [Builder](https://ckeditor.com/ckeditor-5/builder?redirect=docs) to create a new build, or you can update your existing project from the legacy Online Builder. **We recommend using the new interactive Builder**, but if you want to keep your existing build, you can follow the steps below.
+
+1. Follow the steps in the {@link updating/nim-migration/customized-builds Migrating from customized builds} guide.
+	
+2. Once this is done, remove the old `build' folder and run the following command to create a new build of CKEditor&nbsp;5.
+
+		```bash
+		npm run build
+		```
+
+3. There should be three files in the new `build` folder:
+
+			* `ckeditor.d.ts`,
+			* `ckeditor.js`,
+			* `ckeditor.js.map`.
+
+		Now you can start to remove some unused webpack plugins and update the `webpack.config.js` file.
```

Comment:
I'm guessing the suggestion above will resolve the problem with this fragment. The whole thing is rendered as a code snippet, but I'm guessing that wasn't the intention.